lineage/tHMM.py

```diff
@@ -1,15 +1,22 @@
 """ This file holds the parameters of our tHMM in the tHMM class. """
 
 import numpy as np
-import scipy.stats as sp
-from .Lineage_utils import get_numLineages, init_Population
+from .StateDistribution import StateDistribution, tHMM_E_init
 from .tHMM_utils import max_gen, get_gen
 
+class estimate:
+    def __init__(self, numStates):
+        self.numStates = numStates
+        self.pi = np.ones((numStates)) / numStates
```

Comment:
what is the relationsihp between this estimate class and the state_distribution.py parameters

Reply:
estimate class is restricted to tracking the pi, T, and E _estimates_. the StateDistribution object  as well as the LineageTree object holds the _true_ values.